### PR TITLE
FOUR-9666: (QA Observation) Inconsistent top menu highlighting

### PR DESCRIPTION
**Pull request by CarliPinell**
## Issue & Reproduction Steps
(QA Observation) Inconsistent top menu highlighting

## Solution
This PR implements a solution to reported highligh bug in Top Navbar when Analytics-reporting Package is installed, and Top Menu behavior changes when "Analytics" option is added between other Options.

## Related Tickets & Packages
- FOUR-9666

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:deploy

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-3152082894.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-3152082894.eng.processmaker.net

File: resources/views/layouts/navbar.blade.php [hunk 1]
```diff
@@ -64,6 +65,24 @@
...
+                    if ($menuItem['isActive'] !== true) {
+                        $menuItems[$key]['isActive'] = false;
+                    } else {
+                        $falseFlag = false;
+                    }
+                } else {
+                    $menuItems[$key]['isActive'] = false;
+                }
+                // Admin link is different as url()->current() because controller route adds /users
+                if ($menuItems[$key]['title'] === 'Admin' && $falseFlag === true) {
```
Comment (pmPaulis) [inline resources/views/layouts/navbar.blade.php:81]: Is there a way to avoid the harcode value 'Admin'

**Review by pmPaulis — CHANGES_REQUESTED** [review]

File: resources/views/layouts/navbar.blade.php (hunk 1 above)
Comment (CarliPinell) [reply resources/views/layouts/navbar.blade.php:81]: I'm open to hear another option

Comment (CarliPinell) [reply resources/views/layouts/navbar.blade.php]: An improvement was implemented to avoid Hardcode

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-3152082894.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-3152082894.eng.processmaker.net

**Pull request closed without merging**